03_network/on-prem/vpc.tf

```diff
@@ -0,0 +1,17 @@
+module "vpc" {
+  source  = "terraform-google-modules/network/google"
+  version = "7.3.0"
+
+  project_id   = data.tfe_outputs.projects-on-prem.values.net_proj_id
```

Comment:
`project_id   = data.tfe_outputs.net-project-on-prem.values.net_proj_id`

Reply:
You might want to change the name of the output block in the net on prem workspace